### PR TITLE
[release-4.13] Bug OCPBUGS-20017: pmc set boundary hops to 0 and clean up sockets

### PR DESCRIPTION
**Pull request by jzding**

### CODE REVIEW

File: pkg/pmc/pmc.go [hunk 1]
```diff
@@ -2,32 +2,52 @@ package pmc
...
+	}()
+
 	if err = e.Send(cmdStr + "\n"); err == nil {
 		result, matches, err = e.Expect(promptRE, cmdTimeout)
 		if err != nil {
 			glog.Errorf("pmc result match error %s", err)
 			return
 		}
-		err = e.Send("\x03")
+		glog.Infof("pmc result: %s", result)
```
Comment (aneeshkp) [inline pkg/pmc/pmc.go:50]: we can remove the log. Too much log or use glog.debug

Comment (jzding) [reply pkg/pmc/pmc.go]: removed

**Review by aneeshkp — APPROVED** [review]

**Comment by mcornea** [comment]
lgtm

**Pull request merged**